docker-compose.yml

```diff
@@ -13,19 +13,27 @@ services:
     networks:
       - ctf
   django:
-    build: ./backend
+    env_file:
+      - ./backend/.env
+    environment:
+      - MODE=dev
+      - DB_HOST=pgdb
+      - DB_PORT=5432
+    build: 
+      context: ./backend
+      target: base
```

Comment:
`target: base` means that when you docker-compose up only the part of the Dockerfile labeled "base" will run (i.e. not the server)